modules/admin/proc-disabling-telemetry-using-helm.adoc

```diff
@@ -0,0 +1,56 @@
+[id="proc-disabling-telemetry-using-helm_{context}"]
+= Disabling telemetry using the Helm Chart
+
+You can disable the telemetry data collection feature by using the Helm Chart.
+
+.Procedure
+
+. In the *Developer* perspective of the web console, go to the *Helm* view to see the list of Helm releases.
+. Click the *overflow* menu on the Helm release that you want to use and select *Upgrade*.
```

Comment:
@hmanwani-rh, I used click at the first instance because that is an icon with three dots that a user has to click. once the user clicks that icon, 2 options opens in a list from which the user has to select one. So, i think based on the context, i think we can keep the step as is. no need to change. 
even in that thread, Joceylyn mentioned `base your decisions on the content and audience, and apply terms consistently`
in Gaurav's case, the content was for both (mobile and non-mobile devices). This is why, he used select.

Reply:
It is also consistent with already used instances in our admin guide: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-398/admin-rhdh/#proc-rbac-ui-delete-role_assembly-rhdh-observability

Reply:
Sure, that makes sense. Thanks for looking into it, @shipsing.